list.txt

```diff
@@ -55737,4 +55738,4 @@ zzuwnakb.pl
 zzv2bfja5.pl
 zzz.com
 zzzmail.pl
-zzzzzzzzzzzzz.com
\ No newline at end of file
+zzzzzzzzzzzzz.com
```

Comment:
I updated this file from the Github web UI, seems like it changed the last line format. Hope that's fine.

Reply:
Could you revert this change? thanks